crates/neon-runtime/src/napi/date.rs

```diff
@@ -0,0 +1,14 @@
+use nodejs_sys as napi;
+use raw::{Env, Local};
+
+pub unsafe extern "C" fn new_date(env: Env, out: *mut Local, num: f64) {
```

Comment:
I'd probably call the `num` parameter `value`, for consistency with the `value()` method.